custom_event_checks/github/dependabot/dependabot_pr_age.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/sh
+# Fill in your API tokens or add these as secrets:
+#ORG_GITLAB_TOKEN="ADD GITLAB API TOKEN HERE"
+#ORG_GITHUB_TOKEN="ADD GITHUB API TOKEN HERE"
+#OPSLEVEL_INTEGRATION_URL="https://app.opslevel.com/integrations/custom_event/<ADD ENDPOINT TOKEN FROM OPSLEVEL HERE>"
+
+echo "${ORG_GITLAB_TOKEN}" | glab auth login --stdin
+echo "${ORG_GITHUB_TOKEN}" | gh auth login --with-token
+
+query_gitlab() {
+    export ALIAS=${1}
+    PROJECT=${2}
+    QUERY=$(echo "query GetMRs{project(fullPath: \"${PROJECT}\") {mergeRequests(labels: [\"dependencies\"], state: opened) {nodes {webUrl title createdAt}}}}")
+    DATA=$(glab api graphql --paginate -f query="${QUERY}" | jq '{alias: env.ALIAS, open: .data.project.mergeRequests.nodes | map({url: .webUrl, title: .title, age: ((now - (.createdAt | fromdateiso8601)) / 60 / 60 / 24 | round)})}')
+    echo "[opslevel] checking project ${PROJECT}"
+    echo $DATA
+    echo $DATA | curl -s -X POST ${OPSLEVEL_INTEGRATION_URL} -H 'content-type: application/json'  --data-binary @-
```

Comment:
After reading the script, it looks like there are some variables that need to be set for it to work correctly. In this case I recommend using `set -u`. You can also have a check at the top of the file that looks something like:

```
if [[ -z "${OPSLEVEL_INTEGRATION_URL:-}"]]; then
    echo "OPSLEVEL_INTEGRATION_URL is not set"
    exit 1
fi
```

Same goes for `ORG_GITLAB_TOKEN`, `ORG_GITHUB_TOKEN`